source/appModuleHandler.py

```diff
@@ -223,45 +275,35 @@ def cleanup():
 			log.exception("Error terminating app module %r" % deadMod)
 
 
-def doesAppModuleExist(name: str) -> bool:
-	for possibleModName in _getPossibleAppModuleNamesForExecutable(name):
-		modExists = any(
-			importer.find_module(
-				f"appModules.{possibleModName}"
-			) for importer in _importers
-		)
-		if modExists:
-			return modExists
-	return False  # None of the aliases exists
-
-
 def fetchAppModule(processID: int, appName: str) -> AppModule:
 	"""Returns an appModule found in the appModules directory, for the given application name.
 	@param processID: process ID for it to be associated with
 	@param appName: the application name for which an appModule should be found.
-	@returns: the appModule, or None if not found
+	@returns: the appModule.
 	"""
-	# First, check whether the module exists.
-	# We need to do this separately because even though an ImportError is raised when a module can't be found, it might also be raised for other reasons.
 	modName = appName
 
-	if doesAppModuleExist(modName):
-		try:
-			return _importAppModuleForExecutable(modName).AppModule(processID, appName)
-		except:
-			log.exception(f"error in appModule {modName!r}")
-			import ui
-			import speech.priorities
-			ui.message(
-				# Translators: This is presented when errors are found in an appModule
-				# (example output: error in appModule explorer).
-				_("Error in appModule %s") % modName,
-				speechPriority=speech.priorities.Spri.NOW
-			)
+	try:
+		importedMod = _importAppModuleForExecutable(modName)
+		if importedMod is not None:
+			return importedMod.AppModule(processID, appName)
+		# Broad except since we do not know
+		# what exceptions may be thrown during import / construction of the app Module.
+	except Exception:

```

Comment:
```suggestion
	except Exception:
		# Broad except since we do not know
		# what exceptions may be thrown during import / construction of the app Module.
```

Reply:
I'm not sure I follow why this improves things. Usually the comment explaining why something needs to be done precedes the code which is explained i.e. docstring is at the top of the described method etc.

Reply:
I think this is just a personal opinion sorry, couldn't find anything PEP related.
I've found commenting within the scope to be generally more pythonic.
In your example, the doctstring is under the function header,, in this case, it's within the scope of the exception.